Code/Tools/AssetProcessor/native/resourcecompiler/JobsModel.cpp

```diff
@@ -195,19 +195,8 @@ namespace AssetProcessor
                     {
                         return "";
                     }
-                    if (processDuration.hour() > 0)
-                    {
-                        return processDuration.toString("zzz' ms, 'ss' sec, 'mm' min, 'hh' hr'");
-                    }
-                    if (processDuration.minute() > 0)
-                    {
-                        return processDuration.toString("zzz' ms, 'ss' sec, 'mm' min'");
-                    }
-                    if (processDuration.second() > 0)
-                    {
-                        return processDuration.toString("zzz' ms, 'ss' sec'");
-                    }
-                    return processDuration.toString("zzz' ms'");
+
+                    return processDuration.toString("hh:mm:ss.zzz");
```

Comment:
FYI I added a [GetUTCTimestampMilliseconds](https://github.com/o3de/o3de/blob/development/Code/Framework/AzCore/AzCore/std/time.h#L94-L97) function to `<AzCore/std/time.h>`

We may want to add a function there for hours minutes and seconds which is just half of the `GetUTCTimestamp*` function implementation

Reply:
Yeah, we need a duration API not a whole timestamp here.

Reply:
Well it looks like QTime covers this so no need to do anything.
I was just providing a heads up if we needed to output a timestamp without using QT